cpp/ql/src/semmle/code/cpp/commons/unix/Constants.qll

```diff
@@ -15,44 +20,77 @@ int parseOctal(string input) {
     )
 }
 
+/** Gets the number corresponding to the set-user-ID on execute bit in Unix. */
```

Comment:
```suggestion
/** Gets the number corresponding to the "set-user-ID on execute" bit in Unix. */
```

It took me a few tries to parse this sentence. Same problem in the sentence below.

Reply:
I went back and forth quite a few times on what was most readable as well. Adding the quotes definitely improves it!